.travis_setup.sh

```diff
@@ -105,25 +116,20 @@ else
       # xenial
       DB_SYS_USER=root
     fi
-  else
-    # Install from official mysql packages.
-    MYSQL_OPTS=''
-    DB_SYS_USER=root
   fi
+fi
 
-  if [ "${CHANGED_PWD}" = true ]; then
-    # https://www.percona.com/blog/2016/03/16/change-user-password-in-mysql-5-7-with-plugin-auth_socket/
-    sudo mysql ${MYSQL_OPTS} -u "${DB_SYS_USER}" \
-      -e "ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY ''"
-  elif [ "${CHANGED_PWD_BY_RECREATE}" = true ]; then
-    sudo mysql ${MYSQL_OPTS} -u "${DB_SYS_USER}" <<SQL
+if [ "${CHANGED_PASSWORD}" = true ]; then
+  # https://www.percona.com/blog/2016/03/16/change-user-password-in-mysql-5-7-with-plugin-auth_socket/
+  sudo mysql ${MYSQL_OPTS} -u "${DB_SYS_USER}" \
+    -e "ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY ''"
+elif [ "${CHANGED_PASSWORD_BY_RECREATE}" = true ]; then
+  sudo mysql ${MYSQL_OPTS} -u "${DB_SYS_USER}" <<SQL
 DROP USER 'root'@'localhost';
 CREATE USER 'root'@'localhost' IDENTIFIED BY '';
 GRANT ALL PRIVILEGES ON *.* TO 'root'@'localhost' WITH GRANT OPTION;
 FLUSH PRIVILEGES;
 SQL
-  fi
-
-  # IF NOT EXISTS is mariadb-10+ only - https://mariadb.com/kb/en/mariadb/comment-syntax/
-  mysql -u root -e 'CREATE DATABASE /*M!50701 IF NOT EXISTS */ test'
```

Comment:
The new macOS `db: 'mysql@5.6'` case showed the error when running `mysql -u root -e 'CREATE DATABASE /*M!50701 IF NOT EXISTS */ test'`, because it was recognized as `mysql -u root -e 'CREATE DATABASE test'` and there was `test` database already. As every used database supports `IF NOT EXISTS` in CIs, I think we can remove the `/*M!50701 ... */`.